packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -37,16 +38,15 @@ test.describe('severity-2 #smoke', () => {
       await expect(connectAnotherDevice.fxaConnected).toBeEnabled();
     });
 
-    // TODO in FXA-8973 - use sign in not sign up flow
     test('verified, resend', async ({ emails, target, syncBrowserPages }) => {
       const { configPage, page, login, connectAnotherDevice, signinTokenCode } =
         syncBrowserPages;
 
       const config = await configPage.getConfig();
       const [email] = emails;
       test.fixme(
-        config.showReactApp.signUpRoutes,
-        'this test goes through the signup flow instead of sign in, skipping for react'
+        true,
+        'to be fixed to use signin and not signup flow, see FXA-8973'
```

Comment:
Is it better to leave the condition as is, with the JIRA link, so we maintain coverage when it's false? I see @vpomerleau is working on this atm.

```suggestion
        config.showReactApp.signUpRoutes,
        'this test goes through the signup flow instead of sign in, skipping for react, see FXA-8973'
```

Reply:
Sure, no problem. I will change it.